nservicebus/ravendb/manual-dtc-settings.md

```diff
@@ -0,0 +1,57 @@
+---
+title: 'Setting RavenDB DTC settings manually'
+summary: 'Guidance on how to change the RavenDB ResourceManagerID and TransactionRecoveryStorage.'
+tags: [RavenDB, Persistence]
+redirects:
+ - nservicebus/ravendb/how-to-change-resourcemanagerid
+ - nservicebus/ravendb/resourcemanagerid
+---
+
+In order to provide reliable support for distributed transactions in RavenDB, a custom DocumentStore must be provided to NServiceBus and configured to uniquely identify the NServiceBus endpoint to the Distributed Transaction Coordinator (DTC) and provide a storage location for uncommitted transaction recovery.
+
+## Definitions
+
+Support for the Distributed Transaction Coordinator (DTC) in RavenDB is dependent upon the **ResourceManagerId** and **TransactionRecoveryStorage** settings.
+
+### ResourceManagerId
+
+The ResourceMangaerId is a Guid that uniquely identifies a transactional resource on a machine, and must be both unique on that machine, and stable (deterministic) between restarts of the process utilizing the DTC. If more than one RavenDB document store attempts to use the same ResourceManagerId, it can result in the following error during a commit operation:
```

Comment:
@johnsimons Spelling mistake? Which word?

@mauroservienti Do we always need to refer to a document store by its class name? I feel like if it's DocumentStore then it also has to be `DocumentStore` and excessive `CodeInText` style makes it harder to read.